fix_installation.sh

```diff
@@ -89,11 +89,6 @@ disable_npm_audit
 # Enforce strict version checks before installing new packages
 force_strict_npm_version_checks
 
-if [ "$HOST_PLATFORM" = "freebsd" ]; then
-	# Make sure we use the correct python binary
-	set_npm_python
-fi
-
```

Comment:
Can you elaborate why this is no longer necessary?

Reply:
There is no need to do this, as FreeBSD has standard python 3.x installed, see also the commit message, that explains it in detail here the commit message:

For FreeBSD the standard version for python if you install it with `pkg install python` is version 3.7.
If you now try to enforce version 2.7 the installation will fail as this binary does not exist.
I tested it and installation on node depedencies are working fine with python 3.7.
So the gyp problems are fixed already upstream.

I will create another CR to fix the shebang for the two scripts, so I increase the version here not.